compiler/dcalc/from_scopelang.ml

```diff
@@ -132,10 +132,11 @@ let merge_defaults
 
 let tag_with_log_entry
     (e : 'm Ast.expr boxed)
-    (l : log_entry)
-    (markings : Uid.MarkedString.info list) : 'm Ast.expr boxed =
-  let m = mark_tany (Marked.get_mark e) (Expr.pos e) in
-  Expr.eapp (Expr.eop (Log (l, markings)) [TAny, Expr.pos e] m) [e] m
+    (_l : log_entry)
+    (_markings : Uid.MarkedString.info list) : 'm Ast.expr boxed =
+  let _m = mark_tany (Marked.get_mark e) (Expr.pos e) in
+  (* Expr.eapp (Expr.eop (Log (l, markings)) [TAny, Expr.pos e] m) [e] m *)
+  e
```

Comment:
Is this a debug leftover ?

Reply:
(note that we no longer have warn-error in dev mode by default, so you can do this kind of tweak without having to rename all your variables with `_` anymore. And the warnings will remind you that you need to cleanup at some point, and fail in CI as a last resort, so there are only benefits)

Reply:
Nice catch. I need to fix this.

This needs to be enabled only if trace is enabled.

Reply:
done